gemma-core/pom.xml

```diff
@@ -109,6 +109,90 @@
                     </descriptors>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.2</version>
+                <executions>
+                    <execution>
+                        <phase>process-resources</phase>
```

Comment:
The `pre-integration-test` phase could be used for this, although it would be redundant if the model haven't been changed.